.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cfnLint.path": "/home/macobo/.pyenv/shims/cfn-lint"
+}
```

Comment:
might want to add this to `.gitignore`

Reply:
Oops. Didn't intend to commit this. Fixed :facepalm: